CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Working by Forking
+Just head over to the GitHub page and click the "Fork" button. It's just that simple. Once you've done that, you can use your favorite git client to clone your repo or just head straight to the command line:
+
+## Clone your fork to your local machine
+```
+git clone git@github.com:USERNAME/FORKED-PROJECT.git
```

Comment:
means in our case: 
`git clone git@github.com:/vsirotin/si-units.git `
?
Probably it will be clear?